tests/Install-Scripts.Test/AkaMsLinksTests.cs

```diff
@@ -318,13 +318,13 @@ public void LinkCanBeCreatedForGivenRuntime(string channel, string runtime, stri
         }
 
         [Theory]
-        [InlineData("Current", null, "daily", @"https://aka.ms/dotnet/current/dotnet-sdk-")]
+        [InlineData("STS", null, "daily", @"https://aka.ms/dotnet/current/dotnet-sdk-")]
         [InlineData("LTS", null, "signed", @"https://aka.ms/dotnet/LTS/dotnet-sdk-")]
-        [InlineData("Current", "dotnet", "validated", @"https://aka.ms/dotnet/current/dotnet-runtime-")]
+        [InlineData("STS", "dotnet", "validated", @"https://aka.ms/dotnet/current/dotnet-runtime-")]
         [InlineData("LTS", "dotnet", "preview", @"https://aka.ms/dotnet/LTS/dotnet-runtime-")]
-        [InlineData("Current", "aspnetcore", "daily", @"https://aka.ms/dotnet/current/aspnetcore-runtime-")]
+        [InlineData("STS", "aspnetcore", "daily", @"https://aka.ms/dotnet/current/aspnetcore-runtime-")]
         [InlineData("LTS", "aspnetcore", "signed", @"https://aka.ms/dotnet/LTS/aspnetcore-runtime-")]
-        [InlineData("Current", "windowsdesktop", "validated", @"https://aka.ms/dotnet/current/windowsdesktop-runtime-")]
+        [InlineData("STS", "windowsdesktop", "validated", @"https://aka.ms/dotnet/current/windowsdesktop-runtime-")]
```

Comment:
I believe we will need to continue supporting "current" channel since customers are still using it. If we remove tests related to "current" channel, we may break the related scenarios later and wouldn't notice it. I suggest that we keep the "Current" channel tests and add "STS" tests on the top.

Reply:
Maybe we can stop adding new tests with "current" starting with .NET 7.0 though.